### ISSUE TITLE
Does the shape of PatchGAN output influence CycleGAN?

### ISSUE THREAD

**Issue by Aril9** [junyanz/pytorch-CycleGAN-and-pix2pix#599]
The TensorFlow implement of CycleGAN by Harry Yang also use the n=3 discriminator(PatchGAN), and there exists 2 network versions includes tensorflow and your pytorch, which differ in padding selection. 

But what confuses me is that in this code the shape of  PatchGAN output of pytorch version is 35×35 and that of tensorflow is 32×32, both are different with yours. Here is the [CODE](https://github.com/leehomyc/cyclegan-1/blob/54916070dbedc0408a275ead6f14a942327fc061/model.py#L206) (Pytorch) and[ CODE](https://github.com/leehomyc/cyclegan-1/blob/54916070dbedc0408a275ead6f14a942327fc061/model.py#L185) (TensorFlow)

Does the shape of PatchGAN output influence CycleGAN? And what does it indicate or just represent nothing?

**Comment by junyanz** [comment]
It should not make a big difference, as we will average the loss over all the output.  Feel free to try different paddings in your experiments.

**Comment by Aril9** [comment]
@junyanz Thanks for your reply.
